app/presenters/content_item/withdrawable.rb

```diff
@@ -21,6 +29,19 @@ def withdrawal_notice_component
 
   private
 
+    def context_title?
+      context_title.present?
+    end
+
+    def publication_overview?
+      publication_overview.present?
+    end
+
+    def publication_overview
+      overview = (I18n.exists?("content_item.#{schema_name}") == "publication" || "statistical_data_set")
```

Comment:
Could you expand a little on what's going on here to help me understand? I have two questions:

1) Why is statistical_data_set  relevant? It's not a sub type of publication so we never expect to apply this overview/context title behaviour to it.
2) Why are you looking up whether there is a translation for the schema_name ? Don't we want to always apply overview if it's a publication?

It feels like this could be as simple as line 32 - 34 being

```
    def context_title?
      content_item.#{schema_name} == "publication"
    end
    ```
and lines 36-44 being removed, but I expect I'm missing something.